porcupine/default_filetypes.toml

```diff
@@ -421,3 +421,21 @@ tabs2spaces = true
 indent_size = 4
 comment_prefix = '//'
 autoindent_regexes = {dedent = '(return( .+)?|break|continue);', indent = 'this regex matches nothing^'}
+[[Rust.example_commands]]
```

Comment:
These show up as a part of this PR, because you haven't pulled from my master branch in a while. See e.g. https://akuli.github.io/git-guide/pr.html#making-another-pr. The tl;dr is that you need to run:

```
git pull https://github.com/Akuli/porcupine
```

Reply:
I forgot to do it after merging this I will surely do it.